wagtail/admin/views/bulk_action/utils.py

```
@@ -0,0 +1,22 @@
+from django.apps import apps
+
+from wagtail.core import hooks
+
+
+def get_bulk_actions_from_model(app_label, model_name):
+    bulk_actions_list = []
+    bulk_actions = hooks.get_hooks('register_bulk_action')
+    model = apps.get_model(app_label, model_name)
+
+    for action_class in bulk_actions:
```

suggestion: fail loudly if the `action_class` is not a sublcass of `BulkAction`